sql/hive/src/main/scala/org/apache/spark/sql/hive/client/HiveShim.scala

```diff
@@ -456,6 +463,14 @@ private[client] class Shim_v0_12 extends Shim with Logging {
   def listFunctions(hive: Hive, db: String, pattern: String): Seq[String] = {
     Seq.empty[String]
   }
+
+  override def getDatabaseOwnerName(db: Database): String = Utils.getCurrentUserName()
+
+  override def setDatabaseOwnerName(db: Database, owner: String): Unit = {}
```

Comment:
some version of Hive don't support it?

Reply:
Yes，need 0.13 or higher

Reply:
shall we throw exception if it's not supported?

Reply:
These methods are not directly called by users. i.e. when `createDatabase`, we do nothing with hive-0.12